### ISSUE TITLE
Cannot adjust height of an element

### ISSUE THREAD

**Issue by HooHaa-code** [google/automotive-design-compose#643]
I am trying to create a slider widget.  I cannot get the height of the slider fill to adjust based on some fill percentage.  I've attached an example as well as a link to the figma document 

https://www.figma.com/file/G20dairM5eT2yy6XFBhANA/Ex4Google?type=design&node-id=1-3&mode=design&t=JHpgfDy45SZ7FPz6-0

[Ex4Google-slider.zip](https://github.com/google/automotive-design-compose/files/13707700/Ex4Google-slider.zip)

**Comment by timothyfroehlich** [comment]
I believe I've requested access to the file, though Figma's UI doesn't respond when I click the "Request access" button for some reason.

**Comment by timothyfroehlich** [comment]
I just got your invite, can you invite `froeht@google.com` instead?

**Comment by HooHaa-code** [comment]
Looks like you mane need to request access to our organization.  That could be a problem.  Let me know if it is

**Comment by timothyfroehlich** [comment]
Yep, don't thhink that'll work. Can you export the file and upload it here?

**Comment by timothyfroehlich** [comment]
![image](https://github.com/google/automotive-design-compose/assets/5819722/b4f7279d-81f3-444a-985e-5aec29e87c55)

**Comment by timothyfroehlich** [comment]
I need to run for a bit, but take a look at https://github.com/google/automotive-design-compose/blob/feature/rustlayout/integration-tests/validation/src/main/java/com/android/designcompose/testapp/validation/examples/DialsGaugesTest.kt

That's one of our validation examples for a gauge like what you're trying to set up. 

Here's the rest of the documentation for it: https://google.github.io/automotive-design-compose/docs/modifiers/modifiers#dials-gauges-and-progress-bars-dials-gauges

**Comment by HooHaa-code** [comment]
Looks like I can only export a PNG.  Can you create a blank figma file and share it with me?  I can drop the example in there?

I didn't have much luck with the dials and gauges last time i messed with them.  Seemed mike they worked for horizontal bars but not vertical bars.  I can try some more tomorrow

**Comment by timothyfroehlich** [comment]
It doesn't look like I can invite you to a file either. I'll raise the issue internally, we need an effective way for our users to provide reproduction samples of issues.

**Comment by HooHaa-code** [comment]
@timothyfroehlich Can you see if this link works for you?
https://www.figma.com/file/G20dairM5eT2yy6XFBhANA/Ex4Google?type=design&node-id=1%3A3&mode=design&t=OclSBezpvQOWjE5a-1

**Comment by HooHaa-code** [comment]
Actually.... Saving a .fig file is in the save menu not the export menu.  See if this works for you
[Ex4GoogleSlider.zip](https://github.com/google/automotive-design-compose/files/13821638/Ex4GoogleSlider.zip)

git hub won't let me upload a .fig file so I zipped it first

**Comment by timothyfroehlich** [comment]
Oh, nice. I'll take a look when I get a chance.

**Comment by rylin8** [comment]
Assuming that you want a kind of tapered progress bar, I made one in Figma that kind of works, and I've attached it as a .fig file within the zip file. I say kind of works because I believe we have some issues with rotated masks in the current main line code, but it works in a branch where we are working layout changes. We expect to merge this branch into main soon.

Using the attached example, the node named `progressbar` has the progress bar option enabled from our Dials and Gauges plugin. Then, the code sets the value of this by first specifying a `Meter` customization:
```
@Design(node = "progressbar") progressBar: Meter,
```

which is then set to a floating point value between 0 and 100:
```
progressBar = 50F
```

![Screenshot 2024-01-03 at 3 31 24 PM](https://github.com/google/automotive-design-compose/assets/130500397/f02ab944-d3c7-4e0e-9856-77f2c0ebbe63)
[TaperedProgress.zip](https://github.com/google/automotive-design-compose/files/13824348/TaperedProgress.zip)
